src/libstd/backtrace.rs

```diff
@@ -291,6 +291,12 @@ impl Backtrace {
         Backtrace::create(Backtrace::force_capture as usize)
     }
 
+    /// Forcibly captures a disabled backtrace, regardless of environment
+    /// variable configuration.
+    pub fn disabled() -> Backtrace {
```

Comment:
```suggestion
    pub const fn disabled() -> Backtrace {
```

Important if you are interested in using a disabled &amp;'static Backtrace as fallback for a `fn backtrace(&self) -> &Backtrace`.